src/table.ts

```diff
@@ -2049,7 +2016,7 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
  * that a callback is omitted.
  */
 promisifyAll(Table, {
-  exclude: ['family', 'row'],
+  exclude: ['family', 'row', '#readRowsReqOpts'],
```

Comment:
This makes it so our new helper function #readRowsReqOpts works as a normal function and doesn't return a promise.

Reply:
This is fine for now, but depending on what we decide on for this eventually, I want to leave this comment open in case we need to change it later

Reply:
If we pull `#readRowsReqOpts` out of this class like `populateAttemptHeader` then we don't need this anymore and since modifying this `exclude` variable is not preferred, I think this is a good argument for the change you suggested above which is pulling `#readRowsReqOpts` out of this class like `populateAttemptHeader`.